ui/components/app/modals/confirm-remove-account/confirm-remove-account.test.js

```diff
@@ -46,7 +49,7 @@ const mockNonEvmNetwork = {
   network: {
     chainId: MultichainNetworks.BITCOIN,
     rpcPrefs: {
-      blockExplorerUrl: 'https://mempool.space/',
+      blockExplorerUrl: BITCOIN_BLOCK_EXPLORER_URL,
```

Comment:
This comes from https://github.com/MetaMask/metamask-extension/pull/31297

I see that on `main` we're using a hard-coded value here: https://github.com/MetaMask/metamask-extension/blob/32c83a03b613bd82ce12972e6ed186fe4b02e4af/ui/components/app/modals/confirm-remove-account/confirm-remove-account.test.js#L49

We shouldn't be bringing forward a change like this that wasn't made properly on `main`. I'd recommend we keep the hard-coded value that was on `main`, and check with the team on why this wasn't changed on `main`. If they want this on v12.16.0, they can make it on `main` first and we'll cherry pick it in.

Reply:
Likewise for the other changes in this PR related to this constant